plugins/com.google.cloud.tools.eclipse.util/src/com/google/cloud/tools/eclipse/util/ArtifactRetriever.java

```diff
@@ -96,6 +96,7 @@ URL getMetadataUrl(String groupId, String artifactId) {
                 }
               });
 
+  // todo consider a factory to create or build retrievers for each repo
```

Comment:
I think the reason we added the constructor was to allow multiple repos, though we don't seem to be using that yet.